guides/data/git.md

```diff
@@ -1,8 +1,134 @@
-[git](https://git-scm.com/) is the default data source for Cards.
+With the [git plugin](https://github.com/cardstack/cardstack/tree/master/packages/git) for the Card SDK, you can store your card's data as JSON in a local or remote git repository.
+You get an instant, versioned data persistence layer that supports create, read, update, and delete.
 
-git has many advantages, including versioning, the ability to compare current and past states, and powerful rollback capabilities.
+In this guide, we will show how to save Card data to git by default.
+
+## Installing and configuring
+
+Install [`@cardstack/git`](https://github.com/cardstack/cardstack/tree/master/packages/git) in the `cardhost` of your project:
+
+```sh
+cd cardhost
+yarn add --dev @cardstack/git
+```
+
+## Using a local repository by default
+
+While you are first developing a Card, it can be useful to keep the data in a local git repository, on your own computer.
+In this example, we will set up a local repository from scratch, make it the default data source, and save some Card data.
+
+First, create a new directory, outside of your project directory:
+
+```bash
+cd ..
+mkdir project-data
+```
+
+Next, initialize the repository:
+
+```bash
+cd project-data
+git init
+```
+
+In order for data to be saved to a branch, the branch has to exist first.
+Create a file such as `README.md` and commit it to the `master` branch.
+The first commit is what establishes that `master` exists.
+
+```bash
+touch README.md
+git add README.md
+git commit -m "initial commit"
+```
+
+Now that the repository is set up, we need to configure the plugin to
+use it. You will need to know the absolute path to your data repository.
+
+```bash
+pwd
+# prints /the/path/to/your/project-data/
+```
+
+In `cardhost/data-sources/default.js` for your project, set your default data source to git:
+
+```javascript
+{
```

Comment:
we might want to mention that there is an existing [default data source](https://github.com/cardstack/project-template/blob/master/cardhost/cardstack/data-sources/default.js#L72) with type `@cardstack/ephemeral`, and that this should *replace* that one.

Reply:
...or maybe it's assumed that they already know that...?

Reply:
I like to spell out the assumptions anyway. Added!